starting project 2

LinminZ opened the pull request.

Commit:
starting project 2

Commit:
🏯🌴 Checkpoint
./index.html:22199794/392

Commit:
👾🐮 Checkpoint
./index.html:22199794/668
./.glitch-assets:22199794/355

Commit:
🍯🏄 Checkpoint
./.glitch-assets:22199794/850
./index.html:22199794/2907

Commit:
🐺🐧 Checkpoint
./index.html:22199794/438

Commit:
🚎🐦 Checkpoint
./.glitch-assets:22199794/395
./index.html:22199794/109

Commit:
🛫🚐 Checkpoint
./index.html:22199794/1070
./.glitch-assets:22199794/760

Commit:
added content

Commit:
🖇🎒 Checkpoint
./index.html:22199794/3115

Commit:
🏛😎 Checkpoint
./index.html:22199794/1104
./.glitch-assets:22199794/395

Commit:
👰🏑 Checkpoint
./.glitch-assets:22199794/1179
./index.html:22199794/675

Commit:
💡🚿 Checkpoint
./.glitch-assets:22199794/377
./index.html:22199794/4789

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,42 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Your `head` is missing a [proper `title` element](https://mica-web.github.io/learn/html/best-practices#include-a-proper-title-in-your-html-head). This is to add document information to browser windows and tabs and when a user bookmarks a site, among other things. It can include no other HTML or code, but it can include spaces, words or numbers.

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,42 @@
+  </head>
+  <body>
+    <main>
+      <section id="suzhouhe">
```
I would recommend not adding `id`s or class names until you start writing CSS -- This way, you end up only with the names you need.

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,42 @@
+  <body>
+    <main>
+      <section id="suzhouhe">
+        <h1>
+          Suzhou River
+        </h1>
```
For the purposes of this project, your card should start with an `h2` element -- since we're imagining the card is one of many on a page, there would be some other `h1` on the page that is the title for all the cards ("Romantic movies," for example); you don't have to add this `h1` into your code, but you should make choices for your card component knowing that it would exist.

Let me know if this makes sense or if I can help explain this further.

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,42 @@
+        <h3>
+          2000 | romance | 83min | 7.4
+          <a href="https://www.youtube.com/watch?v=3GLEiu-VhvE">
+            <br />
```
You'll want to remove this `br` element and instead control the spacing around things via CSS.

```suggestion
```

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,42 @@
+      "
+          alt="film poster"
+        />
+        <h3>
+          2000 | romance | 83min | 7.4
+          <a href="https://www.youtube.com/watch?v=3GLEiu-VhvE">
+            <br />
+            watch trailer</a
+          >
+        </h3>
```
A heading is like a title of a thing -- a page, a section, etc. While this is all important info, it should either be:
- separate heading elements, or
- wrapped in another HTML element (maybe a list, since these are all attributes of the film?)

But they should not, however, be all bundled up as a single heading element.

There is a reading for the current week about [HTML lists](https://mica-web.github.io/learn/html/lists).

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,42 @@
+        </h3>
+        
+        <p>
+          The story follows the transient lives of four people at the margins of
+          Chinese soci- ety. An anonymous videographer opens the film with a
+          monologue on the eponymous Suzhou River and love. As the film's narra-
+          tor, the videographer is nevertheless never seen aside from his hands,
+          as the camera serves as his first-person perspective for the audience.
+          He starts by telling the story of a romance between himself and
+          Meimei, a performer at a dive bar in Shanghai...
```
It looks like your description copied over some hyphens from the original text, which you should remove here:
```suggestion
          The story follows the transient lives of four people at the margins of
          Chinese society. An anonymous videographer opens the film with a
          monologue on the eponymous Suzhou River and love. As the film's
          narrator, the videographer is nevertheless never seen aside from his hands,
          as the camera serves as his first-person perspective for the audience.
          He starts by telling the story of a romance between himself and
          Meimei, a performer at a dive bar in Shanghai...
```

angeliquejw reviewed (commented):
Hi, Linmin -- I left some feedback about your HTML.

Also, I saw the Watch Trailer image file in your [Glitch project](https://glistening-beneficial-catfish.glitch.me/) -- it's fine to keep that around as a visual for what you're designing for, but you should style the `a` element in your HTML to match those styles and not use the image file for the button itself. We'll talk more in class about why that's important.

Feel free to comment here or reach out via Slack if you have questions about any of this feedback 👍🏻

Commit:
🏄🐿 Checkpoint
./index.html:22199794/61

LinminZ commented:
> Hi, Linmin -- I left some feedback about your HTML.
> 
> Also, I saw the Watch Trailer image file in your [Glitch project](https://glistening-beneficial-catfish.glitch.me/) -- it's fine to keep that around as a visual for what you're designing for, but you should style the `a` element in your HTML to match those styles and not use the image file for the button itself. We'll talk more in class about why that's important.
> 
> Feel free to comment here or reach out via Slack if you have questions about any of this feedback 👍🏻

LinminZ commented:
> Also, I saw the Watch Trailer image file in your [Glitch project](https://glistening-beneficial-catfish.glitch.me/) -- it's fine to keep that around as a visual for what you're designing for, but you should style the `a` element in your HTML to match those styles and not use the image file for the button itself. We'll talk more in class about why that's important.

Commit:
⛑🍢 Checkpoint
./index.html:22199794/832
./styles.css:22199794/828

Commit:
🎊🍯 Checkpoint
./styles.css:22199794/142
./index.html:22199794/34

Commit:
😚👯 Checkpoint
./index.html:22199794/71
./styles.css:22199794/220

Commit:
🚛💝 Checkpoint
./index.html:22199794/50
./styles.css:22199794/444

Commit:
⚡️🍗 Checkpoint
./styles.css:22199794/273
./index.html:22199794/207

Commit:
🍝📯 Checkpoint
./index.html:22199794/1568
./styles.css:22199794/372

Commit:
☕️🚩 Checkpoint
./index.html:22199794/616
./styles.css:22199794/209

Commit:
🍋🚨 Checkpoint
./index.html:22199794/680
./styles.css:22199794/866

Commit:
🚴⛪️ Checkpoint
./index.html:22199794/307

Commit:
👋🏛 Checkpoint
./index.html:22199794/283
./styles.css:22199794/382

Commit:
🍬🐟 Checkpoint
./styles.css:22199794/62
./index.html:22199794/145

Commit:
🧀👏 Checkpoint
./styles.css:22199794/75

Commit:
💼🍄 Checkpoint
./index.html:22199794/347
./styles.css:22199794/17

Commit:
💳❣️ Checkpoint
./index.html:22199794/1257
./styles.css:22199794/930

Commit:
🙇😬 Checkpoint
./styles.css:22199794/561
./index.html:22199794/340

Commit:
🧀🚐 Checkpoint
./index.html:22199794/482
./styles.css:22199794/30

Commit:
⏱🍝 Checkpoint
./index.html:22199794/614
./styles.css:22199794/78

Commit:
🐉🎟 Checkpoint
./styles.css:22199794/511
./index.html:22199794/32

Commit:
🚿😎 Checkpoint
./styles.css:22199794/132
./.glitch-assets:22199794/788
./index.html:22199794/374

Commit:
🎋📻 Checkpoint
./styles.css:22199794/251
./index.html:22199794/98

Commit:
👶😇 Checkpoint
./styles.css:22199794/235
./index.html:22199794/458

Commit:
🛀🍳 Checkpoint
./index.html:22199794/317
./styles.css:22199794/293

Commit:
🛬❤️ Checkpoint
./index.html:22199794/166
./styles.css:22199794/233

Commit:
🗃🌛 Checkpoint
./index.html:22199794/129
./styles.css:22199794/120

Commit:
😘😚 Checkpoint
./index.html:22199794/493
./styles.css:22199794/278

Commit:
📫🕰 Checkpoint
./index.html:22199794/658
./styles.css:22199794/581

Commit:
👸🚊 Checkpoint
./styles.css:22199794/314

Commit:
🦂🏡 Checkpoint
./.glitch-assets:22199794/1161
./index.html:22199794/1289
./styles.css:22199794/293

Commit:
🎑🔦 Checkpoint
./styles.css:22199794/48
./.glitch-assets:22199794/129

Commit:
🎍🌑 Checkpoint
./index.html:22199794/218
./styles.css:22199794/85

Commit:
🌍🚣 Checkpoint
./index.html:22199794/32
./styles.css:22199794/367

Commit:
🎭🙆 Checkpoint
./styles.css:22199794/863
./index.html:22199794/47

Commit:
🍃🙎 Checkpoint
./index.html:22199794/510
./styles.css:22199794/918

Commit:
⛹🍰 Checkpoint
./index.html:22199794/35
./styles.css:22199794/273

Commit:
💏💼 Checkpoint
./styles.css:22199794/307

Commit:
🌖🍀 Checkpoint
./styles.css:22199794/264

Commit:
🌵💶 Checkpoint
./styles.css:22199794/423

Commit:
🏪⛷ Checkpoint
./styles.css:22199794/34

Commit:
💆📮 Checkpoint
./index.html:22199794/3232

Commit:
🌏🐑 Checkpoint
./index.html:22199794/623
./styles.css:22199794/582

Commit:
⛈🛍 Checkpoint
./styles.css:22199794/2273
./index.html:22199794/3099

Commit:
🎐🐙 Checkpoint
./styles.css:22199794/285
